### ISSUE TITLE
[Bug]: when autocompleting it overwrites the string in a weird way even on newest build

### ISSUE THREAD

**Issue by eleijonmarck** [LunarVim/LunarVim#2044]
### Problem description

```
:version
NVIM v0.6.0
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by brew@HMBRW-A-001-M1-004.local
```

new install without any changes to the setting

also updated and pulled the latest:
```bash
cd ~/.local/share/lunarvim/lvim && git pull
lvim +LvimCacheReset +PackerUpdate
```

https://user-images.githubusercontent.com/7727602/145000941-4109cec4-c435-4983-920f-cc63015b8a63.mov

### LunarVim version

0.6.1

### Neovim version (>= 0.5)

0.6.0

### Relevant log output

```shell
nvim -v:checkhealth:messages
NVIM v0.6.0
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by brew@HMBRW-A-001-M1-004.local

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/opt/homebrew/Cellar/neovim/0.6.0/share/nvim"

Run :checkhealth for more info
```

### I have

- [X] Read the readme
- [X] Read the wiki
- [X] Searched for similar issues

**Comment by chase** [comment]
@kylo252 We should probably cut a release to stable soon, since this has popped up more than once on Discord.

This behavior has been fixed in nvim-cmp, but since @eleijonmarck is on version LunarVim 0.6.1, he's missing a lot of fixes.

**Comment by kylo252** [comment]
This is now solved with #2083
